Upstream: Should be
Reason: Fixes compilation by making std::runtime_error available

diff --git a/inference-engine/thirdparty/clDNN/kernel_selector/common/tensor_type.h b/inference-engine/thirdparty/clDNN/kernel_selector/common/tensor_type.h
index 3dbdfd0..2226f1a 100644
--- a/inference-engine/thirdparty/clDNN/kernel_selector/common/tensor_type.h
+++ b/inference-engine/thirdparty/clDNN/kernel_selector/common/tensor_type.h
@@ -25,6 +25,7 @@
 #include <array>
 #include <string>
 #include <utility>
+#include <stdexcept>
 
 namespace kernel_selector {
 #define KERNEL_SELECTOR_TENSOR_DIM_MAX 9
